utilities/misc.js

```diff
@@ -20,6 +20,31 @@ function JSONtoCSV(arr) {
         .join("\n");
 }
 
+function profilesToCsvArray(profiles) {
```

Comment:
it might be helpful to have a comment saying what data structure profiles is expecting